share/spice/timer/timer.css

```diff
@@ -54,7 +54,7 @@
 }
 
 .zci--timer a#add_timer_btn:hover {
```

Comment:
@nalinbhardwaj Could you change this rule back and add a new one that targets dark mode? We have a class called `.dark-bg` that can be used like `.is-mobile`

Basically we don't want the color to change on hover, and this rule exists to override the default link behavior.